voxblox_tango_interface/src/pybind11/bind.cc

```diff
@@ -0,0 +1,14 @@
+#include <pybind11/pybind11.h>
+namespace py = pybind11;
+
+void layer_bind(py::module &);
+void tango_layer_interface_bind(py::module &);
+void tango_layer_io_bind(py::module &);
+
+PYBIND11_MODULE(voxblox_tango_interfacepy, m) {
+  // NOTE(mereweth@jpl.nasa.gov) - this could come from voxblox
```

Comment:
not sure what this note means

Reply:
There's a peculiarity in pybind11 when inheriting from a class. Basically, I end up with two copies of the Layer class, one from the voxblox package, and one from voxblox_tango_interface. I can't convert between them in Python, and sometimes they clash.

This is a (confusing) note to myself, so I'll remove it.